src/styles/blocks/base.css

```diff
@@ -0,0 +1,31 @@
+/* #region fonts */
+@font-face {
+  font-family: Avenir;
+  src: url(../../fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: Avenir-Book;
+  src: url(../../fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(../../fonts/Avenir-Heavy.ttf);
+}
+
+/* #endregion fonts */
+
+/* #region basic styles */
+* {
+  box-sizing: border-box;
+  padding: 0;
```

Comment:
Setting padding to 0 for all elements might cause layout issues, as some elements may depend on default padding. Consider applying zero padding to specific elements where necessary.